ldap/servers/slapd/detach.c

```diff
@@ -163,9 +164,22 @@ detach(int slapd_exemode, int importexport_encrypt, int s_port, daemon_ports_t *
             return 1;
         }
         (void)dup2(sd, 0);
+        /* Lets ignore libaccess printf */
         (void)dup2(sd, 1);
         (void)dup2(sd, 2);
         close(sd);
+#ifdef DEBUG
+        /* But preserve other errors like loader undefined symbols */
+		sprintf(buf, "/tmp/ns-slapd-%d-XXXXXX.stderr", getpid());
```

Comment:
We have `PrivateTmp` enabled by default for our dirsrv service. Perhaps this should be logged in some other place, such as `/run/dirsrv`? Then it would be possible to pick up by our tests.

Reply:
Good point. 
But /run/dirsrv is not writable for non root install and I do not want to have to read the dse.ldif so early to get the install path. 
So I propose to first try in /run/dirsrv then in /tmp in case of failure.
An alternative would be to use /var/tmp